components/conference/Organization.tsx

```diff
@@ -0,0 +1,104 @@
+import { FC } from 'react';
+import { Badge, Col, Image, Row } from 'react-bootstrap';
+import { groupBy } from 'web-utility';
+
+import sponsorData from './../data/SponsorData.json';
+import styles from './Organization.module.less';
+
+const { sponsors, partners } = sponsorData;
+
+const renderLevel = (level: number) =>
+  ['铂金赞助商', '金牌赞助商', '银牌赞助商', '铜牌赞助商'][level] || '礼品赞助';
+
+export const OrganizationInfo: FC = () => (
+  <>
```

Comment:
```suggestion
  <Container>
```
网页有个安全边距，在大屏下面看着比较舒服。